### ISSUE TITLE
Reestruturação do projeto

### ISSUE THREAD

**Issue by lucasppimentel** [lucasppimentel/termometro-eleicoes#10]
Estou planejando e fazendo alguns testes relacionados aos dados e todos os tratamentos que vamos precisar fazer para colocar esse projeto de pé. Vamos precisar desenvolver uma pipeline de ETL grande, provavelmente com um robô de web scraping.
Pensando nisso, acho que precisamos repensar a estruturas das pastas desse projeto.

Normalmente acho que o site e os robôs / ETL seriam feitos em repositórios separados. Mas acho que seria legal se a gente conseguisse colocar tudo nesse repositório porque o projeto como um todo ficaria mais claro, ou mais didático.

Sugestão
-
Pensei em usar uma estrutura assim pra dividir esse projeto:

📁termometro-eleicoes
│
├── 📁site
│   ├── 📁infra
│   ├── 📁pages
│   ├── ...
│   └── package.json
│
├── 📁web-scraper
│   ├── 📁data
│   ├── 📁scripts
│   ├── 📁utils
│   ├── ...
│   └── requirements.txt
│
├── README.md
└── .gitignore

**Comment by joaovcdc** [comment]
Concordo com você que estar tudo no mesmo local é mais centralizado. Vamos prosseguir com essa estrutura de pastas.
